<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-29T04:46:06Z" name="Default suite" finished-at="2022-06-29T04:46:08Z" duration-ms="2917">
    <groups>
    </groups>
    <test started-at="2022-06-29T04:46:06Z" name="Default test" finished-at="2022-06-29T04:46:08Z" duration-ms="2917">
      <class name="com.selenium.day1.ReadWriteExcel">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.selenium.day1.ReadWriteExcel@59cba5a]" started-at="2022-06-29T10:16:06Z" name="beforeTest" finished-at="2022-06-29T10:16:08Z" duration-ms="2812" status="FAIL">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[Invalid file path]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: Invalid file pathat java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)at com.selenium.day1.ReadWriteExcel.beforeTest(ReadWriteExcel.java:47)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:577)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.TestRunner.beforeRun(TestRunner.java:529)at org.testng.TestRunner.run(TestRunner.java:497)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="ReadWrite()[pri:0, instance:com.selenium.day1.ReadWriteExcel@59cba5a]" started-at="2022-06-29T10:16:08Z" name="ReadWrite" finished-at="2022-06-29T10:16:08Z" duration-ms="0" status="SKIP">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[Invalid file path]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: Invalid file pathat java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)at com.selenium.day1.ReadWriteExcel.beforeTest(ReadWriteExcel.java:47)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:577)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.TestRunner.beforeRun(TestRunner.java:529)at org.testng.TestRunner.run(TestRunner.java:497)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ReadWrite -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:com.selenium.day1.ReadWriteExcel@59cba5a]" started-at="2022-06-29T10:16:08Z" name="afterTest" finished-at="2022-06-29T10:16:08Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.selenium.day1.ReadWriteExcel -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
